src/miscellaneous/sendEmailNotification.js

```diff
@@ -0,0 +1,34 @@
+import nodemailer from 'nodemailer'
+import 'dotenv/config'
+
+function sendNotification (message, email) {
+  try {
+    const transporter = nodemailer.createTransport({
```

Comment:
As we have had different sessions as a Team, this transporter does not work for us, so if we have to merge the work you need to use a different method. Please use Mailhog since it works effectively for us all. Instalation of Mailhog can be found [here](https://github.com/mailhog/MailHog/blob/master/docs/RELEASES.md). You dont have to change a lot of codes in your transporter. Just change the port and host. 

![error_sending_email](https://user-images.githubusercontent.com/16195731/158114343-dbbee08e-8ce6-45c8-90d3-d610b0d3167a.jpg)